src/routes/results.js

```diff
@@ -4,12 +4,17 @@ const router = express.Router();
 const pool = require('../models/DbConnection');
 
 router.post('', async (req, res) => {
-  const { result: typeId } = req.body;
-  if (!typeId || Number(typeId) > 12 || Number(typeId) < 1) {
+  const { result: typeId, isNewResult: isNewResult } = req.body;
+  if (!typeId || !Number(typeId) || Number(typeId) > 12 || Number(typeId) < 1) {
     return res.status(406).json({ message: 'wrong type id' });
   }
   // results 테이블에 저장
-  await pool.query(`INSERT INTO results(typeId) VALUES(${typeId})`);
+  if (isNewResult !== "true" && isNewResult !== "false") {
```

Comment:
isNewResult가 boolean이 아니라 string이라면 이게 맞을 텐데 프론트에서 boolean으로 보낸다면 수정이 필요할 거 같아요!
```suggestion
  if (typeof isNewResult !== 'boolean') {
```
isNewResult가 boolean이 아니라면 400 에러를 반환하는 방법은 어떨까 싶기도 합니다! :)